test/controllers/announcement_controller_test.exs

```diff
@@ -1,5 +1,6 @@
 defmodule ConstableWeb.AnnouncementControllerTest do
   use ConstableWeb.ConnCase, async: true
+  import ConstableWeb.ControllerHelper, only: [current_user: 1]
```

Comment:
Is this import needed? I may have missed it, but I didn't see `current_user/1` being used.

Reply:
Ah, I was trying to use this, but found a better way to write the test.